app/controllers/application_controller.rb

```diff
@@ -5,6 +5,16 @@ class ApplicationController < ActionController::Base
   responders :flash
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  helper do
```

Comment:
@vast @RamilGilmanov any thoughts about that https://gist.github.com/timurvafin/816dd58467cbc955feec